code/modules/clothing/clothing.dm

```diff
@@ -16,6 +16,8 @@
 	//Used for hardsuits. If false, this piece cannot be retracted while the core module is engaged
 	var/retract_while_active = TRUE
 
+	var/style = 2 //for sanity, 2 max, 0 min
```

Comment:
Making 2 as base value is clearly a bad call. We better set it to 0 and only specify it clothes we need to be cool.